rest_framework/fields.py

```diff
@@ -1139,7 +1139,7 @@ def to_representation(self, value):
         )
 
         if output_format.lower() == ISO_8601:
-            if (isinstance(value, str)):
+            if isinstance(value, (six.string_types, six.text_type)):
```

Comment:
`string_types` is [enough](https://pythonhosted.org/six/#six.string_types), you don't need the `test_type` here